Sonification/lib/CodapPluginHelper.js

```diff
@@ -552,17 +560,22 @@ class CodapPluginHelper {
             action: 'get',
             resource: `dataContext[${context}].selectionList`
         }).then(result => {
-            let caseIDs = result.values.map(v => v.caseID);
             let selectedItems;
-            if (caseIDs.length) {
-                selectedItems = caseIDs
-                    .map(id => {
-                        // item.id is actually the case ID.
-                        return this.items[context]
-                            .find(item => item && item.id === id)
-                    });
-            } else {
-                selectedItems = this.items[context];
+            if (result.success) {
```

Comment:
Maybe `getStrictlySelectedItems` down below could use the same treatment, where we can short circuit it with:
```js
if (!result.success) return [];
```

Reply:
Seems like a good idea! I added it to `getStrictlySelectedItems`